qutebrowser/javascript/webelem.js

```diff
@@ -188,26 +284,52 @@ window._qutebrowser.webelem = (function() {
             return null;
         }
 
-        return serialize_elem(elem);
+        // Check if we got an iframe, and if so, recurse inside of it
+        const frame_elem = call_if_frame(elem,
+            (frame) => serialize_elem(frame.document.activeElement, frame));
+
+        if (frame_elem === null) {
+            return serialize_elem(elem);
+        }
+        return frame_elem;
```

Comment:
I had to read this a few times until it clicked - maybe check `frame_elem !== null` above (and move it up to the block where `frame_elem` is assigned) to make it clearer?